changelog.txt

```diff
@@ -32,6 +32,7 @@
 * Fix - ACSS: Handle errors and edge cases.
 * Add - Add subscriptions support to Bacs.
 * Update - Add tracks events for payment method settings updates.
+* Dev - Add auto-install/update of Stripe Dev Tools to Docker environment.
```

Comment:
IMO it doesn't need to be in the changelog.

Reply:
This was just to prevent the "Validate Changelog" from failing, but I agree. Fixed in [a5bf43a](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4026/commits/a5bf43a0f822b49f42a535539b648c2a70b817f1).